project.godot

```diff
@@ -57,7 +57,7 @@ _global_script_class_icons={
 [application]
 
 config/name="Godot Oculus Quest Toolkit Demo"
-run/main_scene="res://GameMain.tscn"
+run/main_scene="res://demo_scenes/HandTrackingDemoScene.tscn"
```

Comment:
we should not have the hand tracking demo scene as the main scene.

Reply:
It seems there are a number of these details to look out for when going from the development mode back to release mode.  It's like having to remember to delete all your debug print statements.